### ISSUE TITLE
hierarchical Support 

### ISSUE THREAD

**Issue by hhaim** [mstoyano/kbn_c3js_vis#4]
The component work very good.
I wonder if it possible to add support for hierarchical sub-field like the original chart?
This way it would be possible to create sub chart filtered by term-field

Thanks
Hanoh

**Comment by mstoyano** [comment]
Dear Hanoh, 

Thank you for your comment and sorry for the late reply. 
You are right, support hierarchical data is essential for Kibana visualizations. I think that is possible to implement this feature. I hope I can add it soon to the plugin. 

Best regards,

**Comment by erezkoren** [comment]
I agree it would make it much more useful, we would definitely use it more when it will support sub-buckets
